.gitignore

```diff
@@ -16,3 +18,12 @@ profiles.clj
 dev/user.clj
 .lein-repl-history
 ext/packaging
+
+# Bundler local state files
+Gemfile.lock
+Gemfile.local
+/.bundle/
```

Comment:
Are the leading slashes really necessary here? I'm guessing they aren't causing any harm, but they deviate from the previous rev of the file.